shapeshift/web
# chore: enable osmo staking in local and develop

## Description



* enables osmo staking in dev and develop only
* removes old redundant osmo feature flag from config - not used anywhere

## Notice




## Pull Request Type


## Issue (if applicable)



n/a

## Risk

* relates to the defi section/osmo



## Testing

* check staking with multiple accounts
* check atom staking still works as it's related (cosmossdk)



### Engineering

👆



### Operations

👆



## Screenshots (if applicable)
